### ISSUE TITLE
error: undefined reference to 'curl_easy_cleanup', version 'CURL_OPENSSL_3'

### ISSUE THREAD

**Issue by diggerdu** [Genoil/cpp-ethereum#130]
`/usr/lib/gcc/x86_64-linux-gnu/5/../../../x86_64-linux-gnu/libjsonrpccpp-client.so: error: undefined reference to 'curl_easy_cleanup', version 'CURL_OPENSSL_3'
/usr/lib/gcc/x86_64-linux-gnu/5/../../../x86_64-linux-gnu/libjsonrpccpp-client.so: error: undefined reference to 'curl_global_cleanup', version 'CURL_OPENSSL_3'
/usr/lib/gcc/x86_64-linux-gnu/5/../../../x86_64-linux-gnu/libjsonrpccpp-client.so: error: undefined reference to 'curl_easy_setopt', version 'CURL_OPENSSL_3'
/usr/lib/gcc/x86_64-linux-gnu/5/../../../x86_64-linux-gnu/libjsonrpccpp-client.so: error: undefined reference to 'curl_global_init', version 'CURL_OPENSSL_3'
/usr/lib/gcc/x86_64-linux-gnu/5/../../../x86_64-linux-gnu/libjsonrpccpp-client.so: error: undefined reference to 'curl_easy_init', version 'CURL_OPENSSL_3'
/usr/lib/gcc/x86_64-linux-gnu/5/../../../x86_64-linux-gnu/libjsonrpccpp-client.so: error: undefined reference to 'curl_slist_append', version 'CURL_OPENSSL_3'
/usr/lib/gcc/x86_64-linux-gnu/5/../../../x86_64-linux-gnu/libjsonrpccpp-client.so: error: undefined reference to 'curl_slist_free_all', version 'CURL_OPENSSL_3'
/usr/lib/gcc/x86_64-linux-gnu/5/../../../x86_64-linux-gnu/libjsonrpccpp-client.so: error: undefined reference to 'curl_easy_getinfo', version 'CURL_OPENSSL_3'
/usr/lib/gcc/x86_64-linux-gnu/5/../../../x86_64-linux-gnu/libjsonrpccpp-client.so: error: undefined reference to 'curl_easy_perform', version 'CURL_OPENSSL_3'
`

**Comment by mxbi** [comment]
I have this same issue

**Comment by RakshakTalwar** [comment]
The original project might have some pointers. They've resolved a similar bug in the following commit. 

https://github.com/ethereum/cpp-ethereum/commit/3a4a4ce7a8a85ef5c1fe9e52402c6eee2949d879

Looks like it has to do with libjsonrpccpp

**Comment by paulperry** [comment]
I have the same problem.

@RakshakTalwar I don't think the problem you point to is the same.  I even tried the fix by commenting line 54 of MinerAux.h, as the patch suggests, and it did not fix this problem.

I found that the ./build/ethminer/CMakeFIles/ethmier.dir/link.txt file had references to the wrong libcurl.so (/home/user/anaconda3/libcurl.so ?!) . I removed them and added a -lcurl and was able to link.
